src/contexts/session.context.tsx

```diff
@@ -16,6 +17,9 @@ export function SessionContextProvider(props: PropsWithChildren): JSX.Element {
   const [token, setToken] = useState<string>();
   const { authenticationToken } = useStore();
 
+  const tokenWithFallback = token ?? authenticationToken.get();
+  const isLoggedIn = tokenWithFallback != undefined;
```

Comment:
Verify the token expiration date?

Reply:
good idea. Is this short enough to resolve it in a discussion, otherwise I would create a ticket.